### PR TITLE
Add "start_irrigation" service for Yardian

### PR DESCRIPTION
**Pull request by h3l1o5**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/100257
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/yardian.markdown [hunk 1]
```diff
@@ -26,3 +26,40 @@ There is currently support for the following platform within Home Assistant:
...
+For those Yardian Zone switch which are not given this value, the duration will be set to 6 minutes by default.
+
+```yaml
+# Example of customizing the watering duration
+yardian:
+  duration:
+    switch.yardian_smart_sprinkler_myzone1: 12
+    switch.yardian_smart_sprinkler_myzone2: 24
+    switch.yardian_smart_sprinkler_myzone3: 30
+```
```
Comment (edenhaus) [inline source/_integrations/yardian.markdown]:
```suggestion
```

Yaml configuration is not allowed to be added. See parent PR

**Review by edenhaus — DISMISSED** [review]

File: source/_integrations/yardian.markdown (hunk 1 above)
Comment (h3l1o5) [reply source/_integrations/yardian.markdown]: the suggested change has been addressed.

File: source/_integrations/yardian.markdown [hunk 2]
```diff
@@ -26,3 +26,14 @@ There is currently support for the following platform within Home Assistant:
...
+
+## Services
+
+### yardian.start_irrigation
+
+Start a zone for a given number of minutes. This service accepts an Yardian Zone switch entity and allows a given duration.
+
+| Service Data Attribute | Optional | Description                                           |
+| ---------------------- | -------- | ----------------------------------------------------- |
+| `entity_id`            | no       | The Yardian Zone switch to turn on.                   |
```
Comment (frenck) [inline source/_integrations/yardian.markdown]: This is an entity service, meaning it accepts a target. This also means `entity_id` is actually optional (as one could alternatively target an area or a device).

../Frenck

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/yardian.markdown (hunk 2 above)
Comment (h3l1o5) [reply source/_integrations/yardian.markdown]: got it!

**Review by frenck — APPROVED** [review]
Thanks, @h3l1o5 👍 

../Frenck

**Pull request merged**